packages/block-library/src/cover/edit.js

```diff
@@ -115,6 +118,7 @@ function CoverHeightInput( {
 		<BaseControl label={ __( 'Minimum height of cover' ) } id={ inputId }>
 			<UnitControl
 				id={ inputId }
+				isResetValueOnUnitChange
```

Comment:
Curious what's the purpose of this change?

Reply:
Ah! Various units have set default values. For example, `vh` is `20`,  `px` is perhaps `450`.
When switching units, the value will reset to the defined default value.

This will prevent undesired side effects, like switching `400px` to `vh`. Without this reset, it would jump to `400vh`, which is SUPER tall.

This functionality use to be default for `UnitControl`. Since the recent round of enhancements, it's been disabled by default, since it's use-case is less common.

Reply:
So just for clarity, this is more a fix for some breakage already on master right?

Reply:
Ah yes, that would fix the current breakage 🙈 

I can create a separate PR to expedite that fix (if needed)

Reply:
@youknowriad https://github.com/WordPress/gutenberg/pull/22642

🙏